tests/dummy/app/pods/docs/deploying/template.md

```diff
@@ -0,0 +1,165 @@
+ 2. Set [the `repository` field](https://docs.npmjs.com/files/package.json#repository) of your `package.json`.
+ 3. Commit any outstanding changes you've got on your current branch and push them to GitHub.
+ 4. Run `ember deploy production` and answer "yes" if prompted to create a `gh-pages` branch. **Note**: if your repo already has a `gh-pages` branch, you may want to manually archive the existing content there before deploying.
+
+Once the deploy completes and GitHub has a moment to publish your pages site, if all went well you should see your addon's dummy app live at
+<u>https://**[user]**.github.io/**[repo]**/**[current-branch]**</u>.
+
+Now take a look at the `gh-pages` branch either locally or on GitHub. You should see a layout something like this:
+
+```sh
+├── 404.html
+├── index.html
+├── [current-branch]
+│   ├── assets
+│   ├── index.html
+│   └── ...
+└── versions.json
+```
+
+Let's break down what each of those items is doing.
+ - `index.html` simply redirects from the root of your gh-pages site to `/latest` (more details on that [below](#tag-deploys))
+ - `404.html` contains [some smart redirect logic](https://github.com/rafrex/spa-github-pages) to keep you from having to use `locationType: 'hash'` in your dummy app
+ - `versions.json` contains a manifest of the available versions of your documentation
+ - `[current-branch]` contains all the files from your built docs app
+
+If you were to make a change to your dummy app and run `ember deploy production` again right now, the entry for `[current-branch]` in `version.json` and the entire contents of the `[current-branch]` directory would be replaced with the updated version of your site. Next we'll talk about how to manage keeping published documentation around for multiple versions of your addon.
+
+## Versioning Your Content
+
+Whenever you deploy your documentation site with Addon Docs, it places the compiled application in a subdirectory based on the current state of your git repository. All of this behavior [is customizable](#customizing-deploys), but we expect the out-of-the-box configuration should be a good place to get started.
+
+### Tag Deploys
+
+When you run `ember deploy` at a commit that has a git tag associated with it, the app will wind up in a directory named after that tag. For example, if you've just published version 1.2.3 of your addon (creating tag `v1.2.3` in your git repository), your deployed site will be available at <u>https://**[user]**.github.io/**[repo]**/v1.2.3</u>.
+
+By default, deploying from a tagged commit also places a copy of your app under a special directory called `/latest`. As mentioned above, the `index.html` that Addon Docs sets up at the root redirects to `/latest`, so <u>https://**[user]**.github.io/**[repo]**</u> will always bring developers to the documentation for the most recent stable release of your addon.
```

Comment:
Does this apply if I'm deploying an update to a previous tag to e.g. fix a typo? Or do we guard against updating latest in that case?

Reply:
Since a typo fix likely wouldn't be from a tagged commit, I think this would be a case where you'd manually do something like this (assuming you're re-publishing `v1.2.3`):

```sh
ADDON_DOCS_UPDATE_LATEST=true ADDON_DOCS_VERSION_PATH=v1.2.3 ember deploy production
```

Reply:
sounds good! :shipit: